### ISSUE TITLE
tooltips don't always pop up when you hover over an element

### ISSUE THREAD

**Issue by ellisonch** [prime31/Nez#123]
I noticed this behavior in my own app and reproduced it with the sample code below:

```.cs
// this gets added to the bottom of the SampleScene() constructor in SampleScene.cs of Nez.Samples
var table = new Table();
_table.add(table);

var button1 = new CheckBox("name1", Skin.createDefaultSkin().get<CheckBoxStyle>());
var button2 = new CheckBox("name2", Skin.createDefaultSkin().get<CheckBoxStyle>());
table.add(button1);
table.add(button2);

var tt1 = new TextTooltip("name1", button1, Skin.createDefaultSkin().get<TextTooltipStyle>());
var tt2 = new TextTooltip("name2", button2, Skin.createDefaultSkin().get<TextTooltipStyle>());
tt1.setInstant(true);
tt2.setInstant(true);
canvas.stage.addElement(tt1);
canvas.stage.addElement(tt2);
```
I've attached a gif showing how the tooltips behave.  You should be able to see that if you move the mouse fom button1 to button2, the second tooltip does not pop up.  It does pop up, however, if you move it onto the button from another direction.

The bug is still triggerable even without `setInstant(true)`, it's just easier to see with it turned on.

![tooltipbug](https://cloud.githubusercontent.com/assets/1285287/19206895/2808cca6-8cba-11e6-88c8-82da35f40394.gif)

**Comment by prime31** [comment]
I believe this is just do to tool tips being a singleton. A second tooltip will not show until the animation for the first ends. If you set the animation duration to 0 seconds that would
Probably be the quickest workaround.

**Comment by ellisonch** [comment]
I tried setting `TooltipManager.getInstance().animations = false;`.  It did
not help.  As long as the mouse goes directly from button1 to button2,
without passing over something that isn't tooltip enabled first, the second
tool tip does not show up.

**Comment by prime31** [comment]
Aha. Turns out it was just a simple race condition. Does the mouse out or the mouse over happen first? No one knows.

**Comment by ellisonch** [comment]
The fixes for both this and #122 work perfectly!  Thanks!
